components/email/src/Email.svelte

```diff
@@ -344,6 +344,37 @@
       })
     );
   }
+
+  let currentTrigger, currentTooltip, previousTrigger, previousTooltip;
+
+  function toggleTooltip(e) {
```

Comment:
@heathergh while this works, I think this could have actually been a separate "Tooltip" component. That way all attributes could be managed via reactive variables, including expanded/collapsed states.

Something like:
```svelte
<Tooltip>
  <span slot="symbol"><DropdownSymbol /></span>
  {email_address}
</Tooltip>
```